.vib/oauth2-proxy/goss/goss.yaml

```diff
@@ -0,0 +1,26 @@
+http:
+  http://localhost:{{ .Vars.containerPort }}:
+    status: 403
```

Comment:
A bit of a nitpick, but you could check `http://localhost:{{ .Vars.containerPort }}/ping` so as to receive a `20X` status. I think it would look better compared to the `40X` error.

Reply:
Actually, I think it makes sense to have a `403 - Forbidden` status here. The asset successfully refuses to grant access to the resource unless credentials are provided.

Reply:
IMHO, I do agree the 403 makes sense here.